app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CalendarDetailsFragment.kt

```diff
@@ -93,22 +98,39 @@ class CalendarDetailsFragment : RoundedBottomSheetDialogFragment() {
             descriptionTextView.text = calendarItem.description
         }
 
-        if (calendarItem.url.isEmpty()) {
+        if (!isShownInCalendarActivity) {
+            showInCalendarButtonContainer.visibility = View.VISIBLE
+            showInCalendarButton.setOnClickListener { openEventInCalendarActivity(calendarItem) }
+        }
+
+        if (calendarItem.isEditable && !isShownInCalendarActivity) {
```

Comment:
This is the wrong way around, you're showing the edit buttons in the MainActivity (not working there) and not in the CalendarActivity. Please change.

Reply:
Oopsie ... fixed.